src/e2e-test/features/bigqueryexecute/BQExecute.feature

```diff
@@ -173,3 +173,44 @@ Feature: BigQueryExecute - Verify data transfer using BigQuery Execute plugin
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify 1 records updated in BigQuery table: "bqSourceTable" with query "bqExecuteCountDMLUpdate"
+
+  @BQEXECUTE_SOURCE_TEST @BQ_SINK_TEST @BQ_EXECUTE_SQL @BQExecute_Required
+  Scenario: Verify the successful records transfer from Bigquery(source) to Bigquery(sink) using BQExecute with allow relaxation schema as false.
```

Comment:
Change to: Verify the successful records transfer from Bigquery source to Bigquery sink using insert sql query in BQExecute with rowAsArguments as false

Reply:
changed